brave-spring-beans/src/main/java/brave/spring/beans/AsyncReporterFactoryBean.java

```diff
@@ -0,0 +1,64 @@
+package brave.spring.beans;
+
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.FactoryBean;
+import org.springframework.beans.factory.config.AbstractFactoryBean;
+import zipkin.reporter.AsyncReporter;
+import zipkin.reporter.ReporterMetrics;
+import zipkin.reporter.Sender;
+
+/** Spring XML config does not support chained builders. This converts accordingly */
+public class AsyncReporterFactoryBean extends AbstractFactoryBean<AsyncReporter> {
+  Sender sender;
+  ReporterMetrics metrics;
+  Integer messageMaxBytes;
+  Integer messageTimeout;
+  Integer queuedMaxSpans;
+  Integer queuedMaxBytes;
+
+  @Override public Class<? extends AsyncReporter> getObjectType() {
+    return AsyncReporter.class;
+  }
+
+  @Override protected AsyncReporter createInstance() throws Exception {
+    AsyncReporter.Builder builder = AsyncReporter.builder(sender);
+    if (metrics != null) builder.metrics(metrics);
+    if (messageMaxBytes != null) builder.messageMaxBytes(messageMaxBytes);
+    if (messageTimeout != null) builder.messageTimeout(messageTimeout, TimeUnit.SECONDS);
```

Comment:
Should we change this to millisecond unit? @devinsba mentioned this and I think it makes sense.. @sirtyro ?